docs/ref/csrf.txt

```diff
@@ -92,6 +92,13 @@ protection for your views as outlined above.
     The CSRF token cookie is named ``csrftoken`` by default, but you can control
     the cookie name via the :setting:`CSRF_COOKIE_NAME` setting.
 
+    The CSRF header name is ``HTTP_X_CSRFTOKEN`` by default, but you can control
+    the header name via the :setting:`CSRF_HEADER_NAME` setting.
+
+    This header like any other http headers, before it appears in ``request.META``
```

Comment:
I'm not sure this sentence is correct. The new setting controls request.COOKIES, not request.META.